cantabular/table.go

```diff
@@ -309,3 +542,33 @@ func (it *Iterator) createCSVRow(dims []Dimension, count string) ([]string, erro
 
 	return row, nil
 }
+
+func (it *Iterator) createJsonObject(dims []Dimension, count string) (GetObservationResponse, error) {
+
+	var obsResponse GetObservationResponse
+
+	var obsDimArr []ObservationDimension
+	for i := range dims {
+		var obsDim ObservationDimension
+
+		category, err := it.CategoryAtColumn(i)
+
+		if err != nil {
+			return GetObservationResponse{}, fmt.Errorf("failed to find category at column %d, : %w", i, err)
+		}
+		obsDim.Dimension = it.dims[i].Variable.Label
+		obsDim.DimensionID = it.dims[i].Variable.Name
+		obsDim.Option = category.Label
+		obsDim.OptionID = category.Code
+		obsDimArr = append(obsDimArr, obsDim)
+
+	}
+
+	obsResponse.Dimensions = obsDimArr
+	observation, err := strconv.ParseFloat(count, 32)
+	if err != nil {
+		return GetObservationResponse{}, fmt.Errorf("invalid observation %d, : %w", 0, err)
```

Comment:
Why are you passing 0 here for the digit?

Reply:
The response has now been amended, in the code it was taken from this referred to a row but this is no longer relevant.